docs/guides/examples/aggregate_function_combinators/minSimpleState.md

```diff
@@ -0,0 +1,164 @@
+---
+slug: '/examples/aggregate-function-combinators/minSimpleState'
+title: 'minSimpleState'
+description: 'Example of using the minSimpleState combinator'
+keywords: ['min', 'state', 'simple', 'combinator', 'examples', 'minSimpleState']
+sidebar_label: 'minSimpleState'
+---
+
+# minSimpleState {#minsimplestate}
+
+## Description {#description}
+
+The [`SimpleState`](/sql-reference/aggregate-functions/combinators#-simplestate) combinator can be applied to the [`min`](/sql-reference/aggregate-functions/reference/min)
+function to return the minimum value across all input values. It returns the 
+result with type `SimpleAggregateState`.
```

Comment:
link to SimpleAggregateState and maybe say we simple is useful (i.e. its lighter than full aggregatestate but limited to certain functions)